nsxt/resource_nsxt_vpc_static_routes.go

```diff
@@ -82,7 +82,7 @@ func resourceNsxtVpcStaticRoutes() *schema.Resource {
 		Update: resourceNsxtVpcStaticRoutesUpdate,
 		Delete: resourceNsxtVpcStaticRoutesDelete,
 		Importer: &schema.ResourceImporter{
-			State: nsxtVPCPathResourceImporter,
+			State: getVpcPathResourceImporter("/orgs/[org]/projects/[project]/vpcs/[vpc]/static-routes/[route]"),
```

Comment:
Having these long constant strings within the function call is quite terrible.
I wouldn't ask you to change all of those as it's a pain but maybe it's worthwhile to embed these in the metadata struct somehow or use a constant (especially as we plan to generate this as well).

Reply:
My personal preference is how the code is written today, I would argue its more readable this way, since you don't need to scroll up to see the example. However, I'm fine with rewriting this with a constant. I think we should have same approach for current PR and future generated code.

Reply:
I think that generated code is not supposed to be readable. It will be good to have constants, but if it takes time to implement that, there are worse thing that a long literal - like 500+ files in the same directory :)

Reply:
Done